package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "sonarqube-scanner",
   "description": "SonarQube/SonarCloud Scanner for the JavaScript world",
-  "version": "2.8.2",
+  "version": "2.8.3",
```

Comment:
I think you should not change version

Reply:
please let me know if you prefer that I rollback the version, I supposed you'd like to release this fix asap as it is forcing teams, working behind proxy, to stick to the previous version

Reply:
We usually do this in a separate commit.

Reply:
done